app/models/person.rb

```diff
@@ -289,6 +299,16 @@ def import_remote(remote_people, group, event, member_id)
     end # import_remote
   end # CLASS METHODS
 
+  #
+  # MUTATIONS
+  #
+
+  def update_from_oauth_signup(auth, person_params={})
+    self.class.save_omniauth(EmailAddress.find_by(address: auth.info.email),
```

Comment:
I feel like it would be more clear to just use `Person.save_omniauth` here

Reply:
okay. i prefer to not use `Person`. IMHO `self.class` has very clear semantics. what do we do?

Reply:
:heavy_check_mark: